app/pages/learn/01_tutorial/07_rich_client_apps/02_introduction-javafx-animation.md

```diff
@@ -0,0 +1,245 @@
+It operates on the principle of [WritableValue\<T\>](javafxdoc:WritableValue) which are used across JavaFX.
+It additionally provides a variety of built-in transitions for common effects, support for parallel and sequential transitions, and the ability to handle events upon animation completion.
+I will go through all types of animations, starting with `Animation` and its subclasses `Transition` and `Timeline`, before representing a lower level animation with `AnimationTimer`.
+While `Transition` provides a simpler and more user-friendly way to create animations, `Timeline` offers greater flexibility and is suitable for more complex animations.
+In contrast, `AnimationTimer` is designed for frame-by-frame updates and does not take use of `WritableValue<T>`.
+
+## Animation
+
+The [Animation](javafxdoc:Animation) abstract class provides the core functionality for `Transition` and `Timeline` animations.
+
+An `Animation` consists of multiple properties:
+- The `targetFramerate` is the maximum framerate (frames per s) at which this `Animation` will run.
+- The `currentTime` is the current point in time as a `Duration` of the `Animation`.
+- The `rate` defines the direction and speed at which the `Animation` is expected to be played. It supports both positive and negative numbers.
+- The `cycleCount` defines the number of cycles of this `Animation`. It can't be changed while running and must be positive.
+- The `cycleDuration` is the `Duration` of one cycle of this `Animation`. It is the time it takes to play from start to end of the `Animation` **at the default rate of 1.0**.
+- The `totalDuration` indicates the total duration of this `Animation`, including repeats. It is the result of `cycleDuration * cycleCount` or possibly `Duration.INDEFINITE`.
+- The `delay` is the `Duration` that delays the `Animation` when starting.
+- The `autoReverse` property specifies whether the `Animation` will play in reverse direction on alternating cycles.
+- The `onFinished` event handler is used to define additional behavior when the `Animation` finished.
+- The `status` represents the current state of the `Animation`, possible states are `PAUSED`, `RUNNING` and `STOPPED`.
+
+Additionally, it provides several useful methods, like `play()`, `playFrom(String cuePoint)`, `pause()`, `stop()` and more to control the animations flow.
+A quick look into the JavaDocs provides a great overview of its functionalities.
+
+## Transition
+The [Transition](javafxdoc:Transition) abstract class serves as the foundational class for all transitions, presenting a common form of `Animation`.
+JavaFX provides a variety of built-in transitions for common [Node](javafxdoc:Node) and [Shape](javafxdoc:Shape) properties.
+
+**Note:** By default, all transitions, excluding `ParallelTransition` and `SequentialTransition`, utilize the `Interpolator#EASE_BOTH`.
```

Comment:
Well, I placed it here, because I explicitly set the interpolator to `LINEAR` in the examples after that, but yeah might be better in the interpolator section.